### PR TITLE
Updated files to fix a dashboard level bug for Advanced analytics packages

### PR DESCRIPTION
**Pull request by sodhikirti07**
## Description

Updated files to fix a bug in dashboard filters for Beaconing, Lateral Movement Detection and Data Exfiltration Detection.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- 

## Screenshots

Screenshots are in the comment section

### CODE REVIEW

**Comment by sodhikirti07** [comment]
Screenshots for dashboards with filters:

### Beaconing

image


image


image


### Lateral Movement Detection

image

### Data Exfiltration Detection

image

**Review by susan-shu-c — APPROVED** [review]

File: packages/beaconing/kibana/dashboard/beaconing-5c610e50-2d67-11ee-b990-0d17f5be0d77.json [hunk 1]
```diff
@@ -17,6 +17,11 @@
     "version": 1
   },
   "references": [
+    {
```
Comment (ajosh0504) [inline packages/beaconing/kibana/dashboard/beaconing-5c610e50-2d67-11ee-b990-0d17f5be0d77.json:20]: What index pattern is this?

**Review by ajosh0504 — APPROVED** [review]
LGTM! Just left one comment for my own edification.

File: packages/beaconing/kibana/dashboard/beaconing-5c610e50-2d67-11ee-b990-0d17f5be0d77.json (hunk 1 above)
Comment (sodhikirti07) [reply packages/beaconing/kibana/dashboard/beaconing-5c610e50-2d67-11ee-b990-0d17f5be0d77.json:20]: This is how the dashboard object refer to the filter object of any index. In this case, it's for `ml_beaconing`

File: packages/lmd/elasticsearch/transform/pivot_transform/transform.yml [hunk 2]
```diff
@@ -88,5 +88,5 @@ sync:
     delay: 60s
     field: '@timestamp'
 _meta:
-  fleet_transform_version: 2.1.2
+  fleet_transform_version: 2.1.3
```
[the remark this answers is not in the document]
Comment (sodhikirti07) [reply packages/lmd/elasticsearch/transform/pivot_transform/transform.yml:91]: You're right! We don't need to increment the version here. Thank you for flagging this.

**Comment by sodhikirti07** [comment]
@susan-shu-c Should we include the `unattended` flag while releasing this update for `LMD`? I can leave this PR open until we decide whether to retain this flag or not.

**Comment by susan-shu-c** [comment]
Based on discussion with Kirti: to discuss in team meeting tomorrow with product. Sophie did ask to have the unattended flag here: https://github.com/elastic/security-team/issues/7922#issuecomment-1782549840 so trying to see what is the best way forward

**Comment by mrodm** [comment]
Hi @sodhikirti07, please update your branch with the latest contents from main branch. There was an important PR merged updating the CI pipelines. Thanks!

**Comment by susan-shu-c** [comment]
Hi @SourinPaul , can you follow up on this comment: https://github.com/elastic/security-team/issues/7947#issuecomment-1861847262

Copying here since the `unattended` flag issue is blocking this fix from being merged. The reason is that even if this is merged, it'll still be buggy due to the unattended flag, so we want to make sure a decision is made on the flag before this is shipped.

**Comment by susan-shu-c** [comment]
Closing: pulling Kirti's changes to respective PRs which additionally handle unattended flag Serverless fixes.

Beaconing: https://github.com/elastic/integrations/pull/8832

**Pull request closed without merging**